gapic-common/lib/gapic/grpc/service_stub.rb

```diff
@@ -64,6 +66,9 @@ class ServiceStub
     #   be used for intercepting calls before they are executed Interceptors are an EXPERIMENTAL API.
     # @param channel_pool_config [::Gapic::ServiceStub:ChannelPool::Configuration] The configuration for channel
     #     pool. This argument will raise error when `credentials` is provided as a `::GRPC::Core::Channel`.
+    # @param logger [Logger,:default,nil] An explicit logger to use, or one
```

Comment:
Yes, agreed. A lot of our early development in google-cloud-ruby used the other ordering (type first). I think it might have been because earlier versions of YARD required that ordering. But I definitely prefer name-first now that YARD supports both.